conda.recipe/meta.yaml

```diff
@@ -4,6 +4,8 @@ package:
 
 build:
   number: {{ environ.get('GIT_DESCRIBE_NUMBER', 0) }}
+  {% if environ.get('GIT_DESCRIBE_NUMBER', '0') == '0' %}string: py{{ environ.get('PY_VER').replace('.', '') }}_0
+  {% else %}string: py{{ environ.get('PY_VER').replace('.', '') }}_{{ environ.get('GIT_BUILD_STR', 'GIT_STUB') }}{% endif %}
```

Comment:
One thing I liked about the old solution was that it was dead-simple.  We've oscillated a few times between complex solutions that give us more information but fall down from time to time and simple solutions that give less information but just-work.

I'm fine swinging the other way again but, next time there is a binstar-build issue I'm hoping that you're around to fix it :)